skops/card/_model_card.py

```diff
@@ -205,6 +219,24 @@ def add_plot(self, **kwargs: str) -> "Card":
             self._figure_paths[plot_name] = plot_path
         return self
 
+    def add_metrics(self, metric_values):
+        """Writes metrics to the model card.
+
+        Parameters
+        ----------
+        metric_values: dict
+            Dictionary of metric values.
```

Comment:
I think we should be consistent in terms of API here. `add` accepts kwargs, but here we expect a dict. We should choose only one. WDYT @BenjaminBossan

Reply:
I fixed it :)